### DISCUSSION TITLE
Error ketika menghubungkan envorenment pada visual studio code

### DISCUSSION THREAD

**Discussion by FrengkyLR** [teamalgoritma/community#39] [Q&A] [answered]
kak mau nanya, kenapa saat menjalankan visual studio code tidak bisa jalan dan terdapat error seperti dibawah ini, apakah invorenment yang saya buat tidak bisa tersambung pada visual studio code saya ?
minta solusinya kak, Terimakasih
![vscode error](https://user-images.githubusercontent.com/101881998/162344498-c0b9c159-e9ea-4026-af48-30486b76c308.PNG)

**Comment by Litaa** [comment, marked as answer]
Halo Bapak Frengky,

Perintah 'conda activate' tidak dapat dikenali dikarenakan Bapak masih menggunakan mode PowerShell (PS) pada terminal. Agar dapat berjalan dengan baik, Bapak dapat mengubahnya ke dalam mode Command Prompt (cmd). Berikut adalah lampiran dari step yang dapat Bapak lakukan :
1. Pada sebelah kanan terminal, terdapat icon (+) dan juga icon panah ke bawah
2. Bapak silahkan untuk klik tanda panah ke bawah tsb, kemudian pilih Command Prompt
Ilustrasi gambar : 
![image](https://user-images.githubusercontent.com/6070234/162356609-0de7541c-cdca-4cbd-b18d-44b24ffd500f.png)

Demikian yang dapat kami sampaikan. Semoga membantu :)

**Reply by FrengkyLR** [reply]
Waaah sudah solved, Thanks kak !
